### ISSUE TITLE
System font unable to be set

### ISSUE THREAD

**Issue by TheYellowArchitect** [godotengine/godot#67151]
### Godot version

4.0 beta2

### System information

Artix (Arch Linux without systemd) LARBS

### Issue description

I try to use a systemfont instead of creating/importing a font. I type the font which should exist. I get config error.
![godot-issue-font](https://user-images.githubusercontent.com/25630803/194774147-9e3cbf46-b212-486e-9143-7fbb30d269b4.png)

The warnings in the image are unrelated to this error.

Ah also `print(SystemFont.new().get_font_names())` (ofc while disabling the above label so I can compile) returns `[]` aka empty strings. How to fill it with system fonts?

I don't know how to "enable" font support for Godot, nor if the font I typed is valid. The error could be a newbie mistake from my part (I just want to use a default font), but I have to report it as a bug, just in case it's really a bug, so close this issue if it's not

### Steps to reproduce

Basically create a systemfont, and put an (invalid) text on PackedStringArray[0]

**Comment by Calinou** [comment]
Are you using an official build or a self-compiled Godot binary?

**Comment by TheYellowArchitect** [comment]
I am using official build (beta 2, from https://downloads.tuxfamily.org/godotengine/4.0/beta2/Godot_v4.0-beta2_linux.x86_64.zip)

>As the error message mentions, you need to have Godot linked against fontconfig to be able to use system fonts on Linux.

How to do this?
Also if it's not a bug, feel free to close this issue

**Comment by Calinou** [comment]
@TheYellowArchitect Can you still reproduce this on 4.2.2 or later?

**Comment by Calinou** [comment]
Closing due to lack of response. Please comment if you can still reproduce this bug on the latest Godot version.
